src/main/java/pl/coderstrust/accounting/logic/pdf/PdfService.java

```diff
@@ -0,0 +1,179 @@
+package pl.coderstrust.accounting.logic.pdf;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.PDPageContentStream.AppendMode;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import org.apache.pdfbox.util.Matrix;
+import org.springframework.stereotype.Service;
+import pl.coderstrust.accounting.model.Invoice;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+
+@Service
+public class PdfService {
+
+  public ByteArrayOutputStream generatePdf(Table table, Invoice invoice, String nameFile) throws IOException {
```

Comment:
fileName instead of nameFile will be more natural

Reply:
done